test/test-debuglet.js

```diff
@@ -526,6 +527,88 @@ describe('Debuglet', function() {
       debuglet.start();
     });
 
+    it('should reject breakpoints with conditions when allowExpressions=false',
```

Comment:
I can't find a `test-v8debug.js`, do you mean `test-v8debugapi.js`? The issue is that the logic being testing is in `debuglet.js` so I'm not sure it is testable from `test-v8debugapi.js`.